scripts/update_python.py

```diff
@@ -61,23 +67,56 @@ def make_python_patch(arch):
     os.mkdir(os.path.join('python-embed', 'lib'))
     shutil.move(os.path.join('pywin32', 'PLATLIB'), os.path.join('python-embed', 'lib', 'site-packages'))
 
-    subprocess.check_call(['zip', '-rq', os.path.join('..', out_filename), '.'], cwd='python-embed')
+    check_call(['zip', '-rq', os.path.join('..', out_filename), '.'], cwd='python-embed')
 
     upload_url = upload_base + out_filename
     print('Uploading: ' + upload_url)
     cmd = ['gsutil', 'cp', '-n', out_filename, upload_url]
     print(' '.join(cmd))
-    subprocess.check_call(cmd)
+    check_call(cmd)
 
     # cleanup if everything went fine
     shutil.rmtree('python-embed')
     shutil.rmtree('pywin32')
 
 
-def main():
-    for arch in ('amd64', 'win32'):
-        make_python_patch(arch)
+def build_python():
+    if sys.platform.startswith('darwin'):
+        osname = 'macos'
+        # Take some rather drastic steps to link openssl statically
+        check_call(['brew', 'install', 'openssl', 'pkg-config'])
+        os.remove('/usr/local/opt/openssl/lib/libssl.dylib')
+        os.remove('/usr/local/opt/openssl/lib/libcrypto.dylib')
```

Comment:
do these require root?

Reply:
No, apparently on macOS normaly users have control of /usr/local.

See https://docs.brew.sh/FAQ#why-does-homebrew-prefer-i-install-to-usrlocal

Reply:
Interesting, I did not know that.